amirziai/flatten
# Use module directory so util.py exists inside name space

### Summary
Currently util.py is installed directly into python site-packages which could cause a naming conflict.

### Bug Fixes/New Features
This PR introduces a module directory so that util.py is in the flatten_json namespace.

### How to Verify
Run the tests. Reinstall the module and see that util.py is installed in site-packages/flatten_json/util.py

### Side Effects
Should have no side effects.

### Tests
All existing tests pass. No new tests needed.
